guide/popular-topics/intents.md

```diff
@@ -0,0 +1,104 @@
+
+Permissable gateway intents include:
+- `GUILDS` - Events from this intent include:
+  - `guildCreate`
+  - `guildDelete`
+  - `roleCreate`
+  - `roleUpdate`
+  - `roleDelete`
+  - `channelCreate` (within guilds)
+  - `channelUpdate` (within guilds)
+  - `channelDelete` (within guilds)
+  - `channelPinsUpdate` (within guilds)
+- `GUILD_MEMBERS`¹ - Events from this intent include:
+  - `guildMemberAdd`
+  - `guildMemberUpdate`
+  - `guildMemberRemove`
+- `GUILD_BANS` - Events from this intent include:
+  - `guildBanAdd`
+  - `guildBanRemove`
+- `GUILD_EMOJIS` - Events from this intent include:
+  - `emojiUpdate`
+- `GUILD_INTEGRATIONS` - Events from this intent include:
+  - `guildIntegrationsUpdate`
+- `GUILD_WEBHOOKS` - Events from this intent include:
+  - `webhookUpdate`
+- `GUILD_INVITES` - Events from this intent include:
+  - `inviteCreate`
+  - `inviteDelete`
+- `GUILD_VOICE_STATES` - Events from this intent include:
+  - `voiceStateUpdate`
+- `GUILD_PRESENCES`¹ - Events from this intent include (within guilds):
+  - `presenceUpdate`
+- `GUILD_MESSAGES` - Events from this intent include:
+  - `message`
+  - `messageUpdate`
+  - `messageDelete`
+- `GUILD_MESSAGE_REACTIONS` - Events from this intent include (within guilds):
+  - `messageReactionAdd`
+  - `messageReactionRemove`
+  - `messageReactionRemoveAll`
+  - `messageActionRemoveEmoji`
+- `GUILD_MESSAGE_TYPING` - Events from this intent include (within guilds):
+  - `typingStart`
+- `DIRECT_MESSAGES` - Events from this intent include (within DMs):
+  - `channelCreate`
+  - `message`
+  - `channelUpdate`
+  - `channelDelete`
+  - `channelPinsUpdate`
+- `DIRECT_MESSAGE_REACTIONS` - Events from this intent include (within DMs):
+  - `messageReactionAdd`
+  - `messageReactionRemove`
+  - `messageReactionRemoveAll`
+  - `messageActionRemoveEmoji`
+- `DIRECT_MESSAGE_TYPING` - Events from this intent include (within DMs):
+  - `typingStart`
+¹privileged intent (see [Privileged Intents](#privileged-intents))
+
+:::danger
+At the time of writing (2020/03/23), discord.js does not set default intents, which could potentially break things.
+See [this](https://github.com/discordjs/discord.js/issues/3924) issue for more information.
+:::
+
+:::tip
+Make sure you enable all intents you need for your use case! If you miss one, the corresponding events do not get emitted. For example, If you miss the `GUILDS` intent, your bot won't receive the `guildCreate` event, which is crucial if you implement per-guild command prefixes and create a new database entry when the bot is added to new guilds. If you miss the `GUILD_MESSAGES` intent, your client won't receive messages sent within guilds.
+:::
+
+:::tip
+We strongly recommend you omit the `GUILD_PRESENCES` intent if you don't frequently access the presence data.
+:::
+
+```js
+const { Client } = require('discord.js');
+const client = new Client({ ws: { intents: ['GUILDS', 'GUILD_MESSAGES'] } });
+```
+
+### Privileged Intents
```

Comment:
This section should mention (and maybe even suggest) <https://discord.js.org/#/docs/main/stable/class/Intents?scrollTo=s-NON_PRIVILEGED> as good fallback without needing to provide every single intent.